installation/kubernetes/helm/openclarity/values.yaml

```
@@ -452,6 +452,31 @@ gateway:
     # local endpoints.
     externalTrafficPolicy: Cluster
 
+  ingress:
+    # Be careful when using ingress. As there is no authentication on openclarity, your instance may be accessible.
+    # Make sure the ingress remains internal if you decide to enable it.
```

Hi @bbrala, thanks for putting this together!

This is a great note to include in the chart readme. My suggestion is to prefix the comments that you want to include in the helm docs with `--` so that docs gen can automatically pick them up. You can do the same for other keys too.

```suggestion
    # -- Be careful when using ingress. As there is no authentication on openclarity, your instance may be accessible.
    # Make sure the ingress remains internal if you decide to enable it.
```

After that, you can generate the docs automatically with

```
make gen-helm-docs
```